### ISSUE TITLE
Caching icons

### ISSUE THREAD

**Issue by hanjinliu** [pyapp-kit/pyconify#10]
## ❓ Questions and Help

Hi @tlambert03 , the iconfy support is very nice!

I want to ask if the cached svgs are supposed to be used.
After I ran the following code with internet connection, and ran it later without internet connection, it fails to get the icon even though an svg cache named "mdi_pen-add_color_#..." is created.

```python
from magicgui.widgets import PushButton
btn = PushButton(icon="mdi:pen-add")
btn.show(True)
```

Thanks!

- [x] I have searched the [documentation](https://pyapp-kit.github.io/magicgui/)

**Comment by tlambert03** [comment]
found the issue!  fix needs to happen here in pyconify

**Comment by tlambert03** [comment]
the magicgui usage should be fixed with pyconify v0.1.6.  it's building now, please try again in ~10 mins

**Comment by hanjinliu** [comment]
It's fixed nicely, thanks!
